### DISCUSSION TITLE
Freya - Improvement responsive topbar

### DISCUSSION THREAD

**Discussion by brunoguadagnini** [primefaces/community#2060] [PrimeFaces Templates]
I'm trying to implement other action buttons and selectOneMenu in the Freya template topbar, however Freya doesn't have a properly responsive topbar. I would like to do something like the topbar of the poseidon/ultima templates. Would it be possible?

### Freya
![image](https://github.com/primefaces/community/assets/9530283/badbb693-d99e-4c59-a0ab-7c012e8e37ad)

### Poseidon
![image](https://github.com/primefaces/community/assets/9530283/d1c7afac-7932-47a2-b151-9240fa182d1e)

**Comment by melloware** [comment]
Can you just compare the two and see what needs to be modified in Freya?

**Reply by brunoguadagnini** [reply]
It could, but I don't have the poseidon theme purchased, that makes checking the necessary changes more difficult.
